src/main/java/com/onarandombox/MultiverseCore/configuration/EntryFee.java

```diff
@@ -22,6 +22,8 @@
     @Nullable
     private Material currency;
 
+    private final Material DISABLED_MATERIAL = Material.AIR;
```

Comment:
Please don't make unecessarry field into the heap.

Reply:
I feel this makes make why `Material.AIR` should return null clearer, see what @dumptruckman and @nicegamer7 thinks.

Reply:
The amount of memory needed to store an enum is probably negligible. I think since this improves readability, it's fine how it is.

Reply:
Yea that's what I thought as well.